.github/workflows/validation-coding-standards.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Validation / Coding Standards'
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+    name: PHP
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Composer Install
+        run: composer install --prefer-dist --optimize-autoloader
+
+      - name: PHPCS
+        run: composer lint
+
+  css:
+    runs-on: ubuntu-latest
+    name: CSS
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: php-actions/composer@v5
+
+      - name: Yarn Install
+        run: sudo yarn install
+
+      - name: Run stylelint
+        run: yarn lint:css
+
+  js:
+    runs-on: ubuntu-latest
+    name: JS
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: php-actions/composer@v5
+
+      - name: Yarn Install
+        run: sudo yarn install
```

Comment:
Do not run sudo in Actions. If needed, first setup Node, then install and run yarn.